internal/core/client.go

```diff
@@ -22,6 +22,16 @@ import (
 	"io"
 	"log"
 	"net/http"
+	"runtime"
+)
+
+const (
+	// DefaultCloudURL is default base URL for the services.
+	DefaultCloudURL = "https://cloud.mongodb.com"
+	// ClientName of the v2 API client.
+	ClientName = "go-atlas-sdk-admin"
+	// DefaultUserAgent is default user agent header.
+	DefaultUserAgent = ClientName + "/" + Version + " (" + runtime.GOOS + ";" + runtime.GOARCH + ")"
 )
```

Comment:
[super nit] client.go purpose was to separate HTTP client implementation for auth module. 
For those we can have constants.go just in case we would like to remove client.go along with the legacy code that uses it.